src/static/current-state/components/tabs/HistoryTab.tsx

```diff
@@ -7,14 +7,14 @@ import { QueueItemJoined } from '../QueueItem';
 import { WithAPIData } from '../WithAPIData';
 import { Section } from '../Section';
 
-export type HistoryItemsListProps = {
-  history: Array<HistoryItem>;
-  renderEmpty?: () => JSX.Element;
-  bitbucketBaseUrl: string;
-  items: number;
-  pageLen: number;
-  onPageChange: (page: number) => void;
-};
+// export type HistoryItemsListProps = {
```

Comment:
Is that commented on purpose?

Reply:
It wasn't actually being used as it goes with some other code in that file that was already commented out. Not sure if it still needs to be there.